tests/k8s_env/ksp/ksp_test.go

```diff
@@ -1657,15 +1537,13 @@ var _ = Describe("Ksp", func() {
 
 			// Test 2: access by the owner should be allowed
 
-			sout, _, err = K8sExecInPod(ub3, "multiubuntu",
-				[]string{"bash", "-c", "su - user1 -c 'cat /home/user1/secret_data1.txt'"})
-			Expect(err).To(BeNil())
-			fmt.Printf("OUTPUT: %s\n", sout)
-			Expect(sout).To(ContainSubstring("secret file user1"))
+			AssertCommand(ub3, "multiubuntu", []string{"bash", "-c", "su - user1 -c 'cat /home/user1/secret_data1.txt'"},
+				ContainSubstring("secret file user1"), false,
+			)
 
 			// Test 3: write operation on the file by the owner should also be allowed
-
-			sout, _, err = K8sExecInPod(ub3, "multiubuntu",
+			// No need for AssertCommand here since there is nothing to match
```

Comment:
We should move to AssertCommands anyway to have a streamlined way of Executing, if we need to make changes to the function in future atleast it's going to be at a single place.

Let's create a new Good First Issue to migrate all K8sExec to AssertCommand. We do no need to do this in current PR

Reply:
Marking this resolved for now, but leaving the issue creation to @PrimalPimmy